docs/BROWSER.md

```diff
@@ -75,10 +75,11 @@ There is a script to run VNC server and propagate the display number to the test
 
 Untested pseudo bash example
 
-    docker run -d -P selenium/standalone-firefox-debug > containerId.txt
+    docker run --shm-size=256m -d -P selenium/standalone-firefox-debug > containerId.txt
     export WEBDRIVER_CONTAINER_ID=$(cat containerId.txt)
     export BROWSER=remote-webdriver-firefox
     export REMOTE_WEBDRIVER_URL=http://$(docker port $WEBDRIVER_CONTAINER_ID 4444)/wd/hub
     export JENKINS_LOCAL_HOSTNAME=$(ip -4 addr show docker0 | grep -Po 'inet \K[\d.]+')
     mvn test
 
+It is important to use more that the default 64m of shared memory for firefox to avoid crashes like [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1245239) and [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1338771#c10)
```

Comment:
```suggestion
It is important to use more that the default 64m of shared memory for firefox to avoid crashes like [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1245239) and [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1338771#c10)

```